examples/webgl_math_orientation_transform.html

```diff
@@ -130,18 +135,15 @@
 
 			// generate a random point on a sphere
 
-			spherical.theta = Math.random() * Math.PI * 2;
-			spherical.phi = Math.acos( ( 2 * Math.random() ) - 1 );
-			spherical.radius = 2;
```

Comment:
AFAIK, this code produces a uniform random distribution of points on a sphere. Your code does this not.

Reply:
My changes implemented what your comment says. A uniformly-distributed random point is a special case.